content/blog/2023-01-26-ethereum-classic-course-6-ethereum-classic-is-the-original-chain/index.md

```diff
@@ -0,0 +1,132 @@
+
+This assures a unified way of operating the network and executing smart contracts inside the secure environment of Ethereum Classic.
+
+## 2. A Programming Language
+
+For software programs to be sent and stored in the network, and executed by the EVM, there had to be some programming languages that would be compatible and adapted for a blockchain environment.
+
+Solidity was one of the programming languages that were created to write programs for Ethereum, and remains to this day the most popular one for building dapps on Ethereum and Ethereum Classic.
+
+It is syntactically similar to JavaScript, which is widely used by many developers around the world, but it is adapted so it may work on a decentralized peer-to-peer setting.
+
+## 3. State Transition
+
+State transition is a computing model that Ethereum Classic uses so that the EVM may take inputs, execute programs, and produce new state outputs. This is done on a per block basis.
+
+Bitcoin, in contrast, does not have state transition as account balances are calculated by netting out all the history of transactions that point to each account.
+
+State transition in Ethereum Classic mimics the computing model of normal computers and enables all the participating nodes in the network to behave as a single machine, or global computer.
+
+## 4. The Gas System
+
+Basically, the 120 plus opcodes that work with the EVM have a specific cost in a unit called gas to be executed. When users send their transactions, they need to pay for the corresponding gas for miners to execute them.
+
+This payment serves as compensation to miners and imposes a cost per transaction, which reduces spam significantly.
+
+The gas system also imposes a limit of computation cycles that the ETC miners must use to resolve specific executions. If this limit is reached without results, then miners must stop and return the money to users. This solves what is called the halting problem, which is when machines get stuck when trying to resolve a computation.
+
+## 5. The ETChash Mining Algorithm
+
+One of the unique features of Ethereum Classic is the ETChash mining algorithm.
+
+As Ethereum was using the ETHash algorithm, Ethereum Classic had to differentiate its mining format to defend itself from 51% attacks that came from the Ethereum computing base. This led to the creation of ETChash.
+
+ETChash is a modification of ETHash where one component, the DAG, which is a large file that must be used during the mining process, was reduced in size and its size growth was slowed to permit smaller GPU miners; with GPU card capacities that had lower memories, such as 3GB, 4GB, and 6GB; mine and stay mining in Ethereum Classic for a longer period of time.
```

Comment:
Line 106: Rewrite structure
```suggestion
ETChash is a modification of ETHash wherein one component, a large file called the DAG that must be used during the mining process, was reduced in size and slowed in size growth. This permits miners using GPU cards with a lower amount of memory, such as 3GB, 4GB, or 6GB of VRAM, to mine and stay mining on Ethereum Classic for a longer period of time.
```

Reply:
Currently, the DAG size on ETC is ~3.2GB.

GPUs with less than 4GB of VRAM are already unable to mine ETC.